src/AblyRest.php

```diff
@@ -63,7 +64,7 @@ public function channel( $name, $options = array() ) {
      * @return array Statistics
      */
     public function stats( $params = array() ) {
-        return $this->get( '/stats', $headers = array(), $params );
+        return new PaginatedResource( $this, 'Ably\Models\Stats', $cipher = false, '/stats', $params );
```

Comment:
Don't let this effect this PR, but can you add an issue to change this to PaginatedResult for consistency, see https://github.com/ably/ably-ruby/issues/40